packages/aws-cdk/test/integ/init/test-typescript-versions.sh

```diff
@@ -15,15 +15,22 @@ MIN_SUPPORTED_TS_VERSION=${1:-"3.9"}
 SUPPORTED_TS_VERSIONS=$(node ${integdir}/typescript-versions.js ${MIN_SUPPORTED_TS_VERSION})
 
 for version in $SUPPORTED_TS_VERSIONS; do
-    echo "Testing against TypeScript v$version"
+    header TypeScript v$version
 
     setup
 
+    set -x
+    node --version
+    npm --version
+
     cdk init -l typescript sample-app --generate-only
-    npm pkg delete devDependencies
+    sed '/\"devDependencies\"/,/}/ d; /^$/d' package.json > package.json.new && rm package.json && mv package.json.new package.json
```

Comment:
```suggestion
    sed '/\"devDependencies\"/,/}/ d; /^$/d' package.json -i
```

I think `-i` does it in place?

Reply:
It does, but not on MacOS. I thought there's value in being able to run the script locally.